lib/tropic/tropic.flow

```diff
@@ -512,14 +512,18 @@ TDynamicGroup(stackChanges : DynamicBehaviour<[TGroupChange]>) -> TDynamicGroup2
 		maxX : Transform<double>,
 		// How much is allocated to this element, including any y-fill expanders here?
 		maxY : Transform<double>,
-
+		// Text flow direction, it is convenient to use bool
+		// while two alternatives only, otherwise to replace
+		// with union everywhere, including ALL suffixed/prefixed
+		// with LTR/RTL.
+		isRTL : bool,
```

Comment:
We don't use all capital letters, so it should be isRtl

Reply:
Renamed in segregated branch.